_source/_posts/2020-04-29-master-filestream-in-csharp-aspnetcore.md

```diff
@@ -302,6 +318,8 @@ I'm going to share an anecdote with you that explains a particular time this was
 
 Today we've learned how to break down large tasks into small chunks and how to build a secure streaming service. I hope this has given you a starting place for working with some lower-level parts of the .NET framework. They can seem scary but, if you just think about them as reading and writing data, it gets a lot less daunting.
 
+A link to the Github repository for this tutorial can be found [here](https://github.com/chaseaucoin/Okta.Blog.FileStreams). Happy coding!
```

Comment:
We usually fork these to our repo and add an `okta-` prefix and an `example` suffix. That way people can search for them and they show up at https://toolkit.okta.com/apps/. For this one, I'd name it `okta-csharp-filestreams-example`. This also gives you the ability to easily update the README and add 1) a link to the blog post, 2) steps to clone the repo and configure it for Okta.

Reply:
Done!